include/zephyr/net/wifi.h

```diff
@@ -372,23 +372,21 @@ static const char * const wifi_twt_negotiation_type2str[] = {
 
 /** Wi-Fi Target Wake Time (TWT) setup commands. */
 enum wifi_twt_setup_cmd {
-	/** TWT Requests */
-	/* TWT setup request */
+	/** TWT setup request */
```

Comment:
If you think it's still relevant to have them, you could keep the original intent of grouping requests and responses using the following Doxygen construct:

```
/**
 * @name TWT requests
 * @{
 */

...

/**
 * @}
 */

Reply:
Sure, but rethinking, it wasn't adding much value, so, removed it altogether.

Reply:
fair enough :)